src/applications/hca/helpers.jsx

```diff
@@ -25,13 +25,69 @@ export {
   medicalCenterLabels,
 } from 'platform/utilities/medical-centers/medical-centers';
 
+/*  clean address so we only get address related properties 
+    then return the object as JSON so we can match them
+*/
+const cleanAddressObject = address => {
```

Comment:
This is a function that I added that destructures the profile address and only takes data that matches the schema.